pkg/heartbeat/heartbeat.go

```diff
@@ -19,6 +19,7 @@ type Heartbeat struct {
 	CursorPosition      *int       `json:"cursorpos"`
 	Dependencies        []string   `json:"dependencies"`
 	Entity              string     `json:"entity"`
+	EntityRaw           string     `json:"-"`
```

Comment:
I don't think we're populating this with a value anywhere yet?

Reply:
At `remote.WithDetection()`?

Reply:
Somehow didn't see it before.